apps/charterafrica/src/components/Footer/Footer.js

```diff
@@ -17,7 +20,17 @@ const Footer = React.forwardRef(function Footer(props, ref) {
     logo,
     projectDescription,
     siteDescription,
+    newsletter,
   } = props;
+
+  const [toEmbed, setToEmbed] = useState("");
+
+  const embedCode = newsletter?.embedCode;
+
+  useEffect(() => {
+    setToEmbed(embedCode);
+  }, [embedCode]);
+
```

Comment:
```suggestion
```

I've completely removed this code block and everything worked fine. Are you 💯 it adds anything?

Reply:
The error
<img width="369" alt="image" src="https://user-images.githubusercontent.com/39160236/213991625-043699da-e47a-4799-9cdf-e9e203d7511a.png">